frontend/src/views/Dashboard.vue

```diff
@@ -58,6 +58,9 @@ export default {
       }
     }
   },
+  async created() {
+    this.showSkeleton()
```

Comment:
The dashboard renders in 30 ms, I think this would look like a glitch, so let's please remove it.